File: Venmo/src/main/java/com/braintreepayments/api/VenmoInternalCallback.java
package com.braintreepayments.api;

import androidx.annotation.Nullable;

/**
 * Callback for receiving result of
 * {@link VenmoClient#tokenize(VenmoPaymentAuthResult, VenmoTokenizeCallback)}.
 */
interface VenmoInternalCallback {

    /**
     * @param venmoAccountNonce {@link VenmoAccountNonce}
     * @param error an exception that occurred while processing a Venmo result
     */
    void onResult(@Nullable VenmoAccountNonce venmoAccountNonce, @Nullable Exception error);
}

File: Venmo/src/main/java/com/braintreepayments/api/VenmoInternalCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of [VenmoClient.tokenize].
 */
internal fun interface VenmoInternalCallback {

    /**
     * @param venmoAccountNonce [VenmoAccountNonce]
     * @param error an exception that occurred while processing a Venmo result
     */
    fun onResult(venmoAccountNonce: VenmoAccountNonce?, error: Exception?)
}
